pandas/io/tests/generate_legacy_pickles.py

```diff
@@ -143,16 +143,14 @@ def write_legacy_pickles():
     except:
         import pickle
 
-    sys_version = version = pandas.__version__
+    version = pandas.__version__
```

Comment:
The condition looks incorrect in current master and the message is not displayed properly. After this PR, it only need 1 arg (sys.argv must be 2).

```
# current master
$ generate_legacy_pickles.py 1.1
Traceback (most recent call last):
  File ".../generate_legacy_pickles.py", line 151, in write_legacy_pickles
    output_dir = str(sys.argv[2])
IndexError: list index out of range
```